src/events/crawler/scrapers/RUS/index.js

```diff
@@ -0,0 +1,65 @@
+import needle from 'needle';
+
+import populations from './populations.json';
+
+// Remap some of the mismatching or ambiguous names
+const POPULATION_REMAP = {
+  'Архангельская область': 'Архангельская область без Ненецкого автономного округа',
+  'Тюменская область': 'Тюменская область без автономных округов',
+  'Ханты-Мансийский АО': 'Ханты-Мансийский автономный округ-Югра'
+};
+
+const scraper = {
+  country: 'RUS',
+  aggregate: 'state',
+  url: 'https://yandex.ru/maps/api/covid?csrfToken=',
+  type: 'json',
+  curators: [{ name: 'Yandex', url: 'https://yandex.ru' }],
+  sources: [
+    {
+      description:
+        'Data provided by Rospotrebnadzor (Federal Service for Surveillance on Consumer Rights Protection and Human Wellbeing)',
+      name: 'Rospotrebnadzor',
+      url: 'https://www.rospotrebnadzor.ru/'
+    }
+  ],
+  maintainers: [
+    {
+      name: 'Arseniy Ivanov',
+      twitter: '@freeatnet',
+      github: 'freeatnet',
+      email: 'arseniy+coronadatascraper@freeatnet.com'
+    }
+  ],
+  async scraper() {
+    const csrfRequestResponse = await needle('get', this.url, {}, { parse_response: true });
+    const csrfCookies = csrfRequestResponse.cookies;
+    const { csrfToken } = csrfRequestResponse.body;
+    // assert typeof csrfToken !== 'undefined'
+
+    const { body } = await needle('get', `${this.url}${csrfToken}`, { cookies: csrfCookies, parse_response: true });
```

Comment:
We need the scraper to use `fetch.js` for caching to work. If you look at `fetch()` in `fetch.js`, it accepts an options object. I would suggest passing cookies in that options object, and updating this [line](https://github.com/lazd/coronadatascraper/blob/f86a1193e5564af634a2dbe7637be2c71d621185/src/events/crawler/lib/get.js#L64) to add cookies to the needle request.